src/internal/clusterstate/v2.8.0/pfsdb.go

```diff
@@ -601,12 +575,16 @@ func migrateBranches(ctx context.Context, env migrations.Env) error {
 			ELSE
 				row := NEW;
 			END IF;
-			payload := TG_OP || ' ' || row.int_id::text;
+			payload := TG_OP || ' ' || row.id::text;
```

Comment:
What happened to int_id?

Reply:
this is in the branches code (which only has an 'id'). We're going to re-add the trigger code and call it as part of the last PR which updates all the callsites.